website/docs/Configuration.md

```diff
@@ -102,6 +102,34 @@ collapsed = false # Determines with the section grouping is collapsed or expande
 classes = ["Class7", "ClassAte", "Class9"]
 ```
 
+### Table of Contents (TOC)
+
+You can also customize the categories in each LuaClass page's table of contents (TOC). To add a category, first add it as a `tag` field within a doc comment for a class's function.
```

Comment:
Here we are using `LuaClass`, which is an internal name, to refer to something the user thus far only knows as an "API page". We should use this terminology instead.

```suggestion
You can customize categories in the Table of Contents of each API page. To create a category, first tag all items that should be included in the category with the `@tag` tag.
```